public/stylesheets/style.css

```diff
@@ -13,18 +13,6 @@ form.login {
     font-weight: bold;
 }
 
-.row {
```

Comment:
Ed why are you removing this section?

Reply:
Which section?  Line Nos.?  I didn't remove anything from CSS on git pull from yesterday.  I changed footer class for sign up and login to separate it from the main.handlebars.